cscore/src/main/native/linux/UsbCameraImpl.cpp

```diff
@@ -108,6 +108,11 @@ static bool IsPercentageProperty(wpi::StringRef name) {
 static constexpr const int quirkLifeCamHd3000[] = {
     5, 10, 20, 39, 78, 156, 312, 625, 1250, 2500, 5000, 10000, 20000};
 
+static constexpr char const* quirkPS3EyePropExAuto = "auto_exposure";
+static constexpr char const* quirkPS3EyePropExValue = "exposure";
```

Comment:
The rest of the codebase uses the style `const char*` instead of `char const*`.

Reply:
In that same file, it uses `char const*` on lines 45-50. Should it be changed to match there as well?

Reply:
Whoops, you're right.  You can keep these as-is.